pkg/collector/upgrade/versions.go

```diff
@@ -77,6 +77,10 @@ var (
 			Version: *semver.MustParse("0.56.0"),
 			upgrade: upgrade0_56_0,
 		},
+		{
+			Version: *semver.MustParse("0.57.0"),
```

Comment:
The release version across the PR is specified as `0.57.2`, any specific reason why we are adding it here as `0.57.0`?

Reply:
I mixed up the versions.